cmake/tribol-config.cmake.in

```diff
@@ -34,54 +34,63 @@ if(NOT TRIBOL_FOUND)
   # Config options
   set(TRIBOL_USE_SINGLE_PRECISION "@TRIBOL_USE_SINGLE_PRECISION@")
   set(TRIBOL_USE_64BIT_INDEXTYPE  "@TRIBOL_USE_64BIT_INDEXTYPE@")
-  set(TRIBOL_ENABLE_SIDRE         "@TRIBOL_ENABLE_SIDRE@")
-  set(TRIBOL_ENABLE_SLIC          "@TRIBOL_ENABLE_SLIC@")
+
+  # TPLs
+  set(TRIBOL_USE_UMPIRE "@TRIBOL_USE_UMPIRE@")
   
   #----------------------------------------------------------------------------
   # Bring in required dependencies for this axom configuration
   #----------------------------------------------------------------------------
   include(CMakeFindDependencyMacro)
 
+  # Enable various find commands to look in non-default paths
+  set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIB32_PATHS TRUE)
+  set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS TRUE)
+  set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIBX32_PATHS TRUE)
+
+  # Tribol is optional
```

Comment:
I think this should be `Umpire is optional`

Reply:
I'm not opposed to including a disclaimer (Tribol may cause high blood pressure, hair loss, etc.), but this probably isn't the right place for it.